pipelines/e2e/e2e.yaml

```diff
@@ -32,10 +32,6 @@ spec:
     - name: enterpriseContractPolicy
       type: string
       description: JSON representation of the EnterpriseContractPolicy
-    - name: enterpriseContractPublicKey
-      type: string
-      description: Public key to use for validation by the enterprise contract
-      default: k8s://openshift-pipelines/public-key
```

Comment:
What if we set this to an empty string? I think it should still cause the value from ECP to be used. This would allow someone to use this Pipeline parameter to over other use cases.

Reply:
It it's only for users to set the parameter, as @ralphbean said, that is not supported. Parameters in release pipelines are supplied by the operator itself. We control user parameters through the `data` key in the release resources.

I think that removing the parameter is the right call here. I think I remember that key not having a default when we started producing the pipelines (in fact, the parameter was there one year ago at least). That must be the only reason we never removed it or did anything with it.

I also remember @gbenhaim being interested in changing the value for anything he was doing not so long ago, so he could be interested in this change.